extensions/default/src/DicomJSONDataSource/index.js

```diff
@@ -206,6 +206,9 @@ function createDicomJSONApi(dicomJsonConfig) {
                 imageId: instance.url,
                 ...series,
                 ...study,
+                // We need to force this flag to false in case the client wants to load the metadata from the image
+                // directly with automaticallyLoadDicomMetadata
+                imageMetadataLoaded: false
```

Comment:
how about only make it false if only series/studies/sop UID are provided. For other use cases where we have the full metadata in the JSON this doesn't makes sense to be false and later in the provider it checks and reads again etc.

Reply:
If you follow the suggestions further in this, then you can figure out when to load appropriately just by which loader it is calling, so there wouldn't be any setting here at all.